process_report/invoices/billable_invoice.py

```diff
@@ -115,6 +117,31 @@ def export_s3(self, s3_bucket):
         super().export_s3(s3_bucket)
         s3_bucket.upload_file(self.old_pi_filepath, self.PI_S3_FILEPATH)
 
+    def _filter_partners(self, data):
+        active_partnerships = list()
+        institute_list = util.load_institute_list()
+        for institute_info in institute_list:
+            if partnership_start_date := institute_info.get(
+                "mghpcc_partnership_start_date"
+            ):
+                partnership_year_month = "{:%Y-%m}".format(
+                    datetime.strptime(partnership_start_date, "%Y-%m-%d")
```

Comment:
@knikolla I think that was because of my concerns described in this comment: https://github.com/CCI-MOC/process_csv_report/pull/96#discussion_r1761622405

Reply:
but yeah, if it's just year and month then it could probably be passed directly.

Reply:
The date format of the partnership start date will now be YYYY-MM. I've updated the PR message accordingly.